src/workingtitle-vcockpits-instruments-navsystems-g3000/html_ui/WorkingTitle/Pages/VCockpit/Instruments/NavSystems/AS3000/MFD/AS3000_MFD.js

```diff
@@ -18,9 +18,9 @@ class AS3000_MFD extends NavSystem {
             ]),
         ];
         
-        Include.addScript("/JS/debug.js", function () {
-            g_modDebugMgr.AddConsole(null);
-        });
+        //Include.addScript("/JS/debug.js", function () {
+        //    g_modDebugMgr.AddConsole(null);
+        //});
```

Comment:
There's not any particular need to do this, it has nil effect if the user doesn't have the devkit installed.   But it doesn't hurt, either and it may keep people who have it for other reasons from being bothered in this one.